### PR TITLE
GTK+ with foss2016a

### PR DESCRIPTION
**Pull request by RvDijk**

### CODE REVIEW

**Comment by RvDijk** [comment]
Dependency on #3350

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2689.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/baf0908ebf22c67f91b6e2e5c536924b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2034.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/4beab6e864ed2421e5947aad9814f45a for a full test report.

File: easybuild/easyconfigs/l/libjpeg-turbo/libjpeg-turbo-1.5.0-foss-2016a-NASM-2.12.02.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+nasmver = '2.12.02'
+versionsuffix = '-NASM-%s' % nasmver
```
Comment (boegel) [inline easybuild/easyconfigs/l/libjpeg-turbo/libjpeg-turbo-1.5.0-foss-2016a-NASM-2.12.02.eb]: @RvDijk there's no need for this, the existing `libjpeg-turbo-1.5.0-foss-2016a.eb` already depends on `NASM` v2.12.02?

File: easybuild/easyconfigs/g/Gdk-Pixbuf/Gdk-Pixbuf-2.35.1-foss-2016a.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+
+dependencies = [
+    ('GLib', '2.48.0'),
+    ('libjpeg-turbo', '1.5.0', '-NASM-2.12.02'),
```
Comment (boegel) [inline easybuild/easyconfigs/g/Gdk-Pixbuf/Gdk-Pixbuf-2.35.1-foss-2016a.eb]: @RvDijk should be fine to just drop the `NASM-*` versionsuffix here

**Comment by boegel** [comment]
@RvDijk requires sync with `develop` since #3541 got merged

**Comment by boegel** [comment]
@RvDijk removing the GTK+ easyconfig doensn't resolve the conflict... To sync with `develop`, I usually just `git merge develop` in the branch, like this: https://github.com/rug-cit-ris/easybuild-easyconfigs/pull/4

Note that the existing easyconfig for `Gdk-Pixbuf` in `develop` depends on `('libjpeg-turbo', '1.4.2', '-NASM-2.12.02')`, so better stick to that?

**Comment by boegel** [comment]
@RvDijk this is now an empty PR?

**Comment by RvDijk** [comment]
@boegel it seems that all modules are already added 7 days ago. This PR can be closed?

**Pull request closed without merging**